languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -27,10 +27,10 @@ object ImportResolver {
     directoryResolver(LanguageFactoryUtil.validateImportsDirectory(zippedImports).toEither)
   }
 
-  def directoryResolver(directory: Path): ImportResolver = CheckedAtoB.fromErrorOr { path =>
+  def directoryResolver(directory: Path, allowEscapingDirectory: Boolean = false): ImportResolver = CheckedAtoB.fromErrorOr { path =>
     Try(Paths.get(directory.resolve(path).toFile.getCanonicalPath)).toErrorOr flatMap { absolutePathToFile =>
       val absolutePathToImports = Paths.get(directory.toJava.getCanonicalPath)
-      if (absolutePathToFile.startsWith(absolutePathToImports)) {
+      if ((!allowEscapingDirectory && absolutePathToFile.startsWith(absolutePathToImports)) || allowEscapingDirectory) {
```

Comment:
I'm pretty sure that the `(!allowEscapingDirectory &&` at the start is redundant. Though if you want it for the short circuiting, maybe make the whole expression `if (allowEscapingDirectory || absolutePathToFile...) {`

Reply:
It was for the short-circuiting, but this is better.

Reply:
| x | y | x \|\| y | (!y && x) \|\| y |  
|---|---|:--------:|:----------------:|
| F | F | F      | F              |  
| F | T | T      | T              |   
| T | F | T      | T              |   
| T | T | T      | T              |

Reply:
good 'ol boolean algebra